src/Microsoft.Identity.Web/TokenAcquisition.cs

```diff
@@ -570,11 +563,34 @@ private IAccount GetAccountByUserFlow(IEnumerable<IAccount> accounts, string use
             foreach (var account in accounts)
             {
                 string accountIdentifier = account.HomeAccountId.ObjectId.Split('.')[0];
-                if (accountIdentifier.EndsWith(userFlow.ToLower()))
+                if (accountIdentifier.EndsWith(userFlow.ToLower(CultureInfo.CurrentCulture), StringComparison.InvariantCulture))
```

Comment:
Please use InvariantCulture in ToLower().

Reply:
yes thanks.

---
In reply to: [420317657](https://github.com/AzureAD/microsoft-identity-web/pull/144#discussion_r420317657) [](ancestors = 420317657)